core/vibe/core/drivers/libevent2.d

```diff
@@ -60,34 +60,53 @@ version (Windows)
 else
 	version(VibePragmaLib) pragma(lib, "event");
 
-version(Windows) {
+version(Windows)
+{
 	import core.sys.windows.winsock2;
 	alias EWOULDBLOCK = WSAEWOULDBLOCK;
-} else version(OSX) {
+}
+else version(OSX)
+{
 	static if (__VERSION__ < 2077) {
 		enum IP_ADD_MEMBERSHIP = 12;
 		enum IP_MULTICAST_LOOP = 11;
 	}
 	else
 		import core.sys.darwin.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
-} else version (FreeBSD) {
+}
+else version (FreeBSD)
+{
 	static if (__VERSION__ < 2077) {
 		enum IP_ADD_MEMBERSHIP  = 12;
 		enum IP_MULTICAST_LOOP  = 11;
 	}
 	else
 		import core.sys.freebsd.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
-} else version (linux) {
+}
+else version (DragonFlyBSD)
+{
+	static if (__VERSION__ < 2077) {
```

Comment:
Somehow the comment of @Geod24 is shown as outdated, but he is right, this should be `__VERSION__ < 2079` (or even later, if `core.sys.dragonflybsd` doesn't get merged into druntime master before 2.079)

Reply:
[See](https://github.com/vibe-d/vibe.d/pull/2028#discussion_r161077152)
No need to check the compiler version. It's like a chicken/egg issue, without compiler support there is not way to get to this code. When compiler support lands, then core.sys.dragonflybsd.netinet.in_ will exist.